bolt/outpost/settlement/v1/settlement.proto

```diff
@@ -178,6 +178,10 @@ message BalanceUpdate {
   uint64 height = 6;
   // total_shares: Total amount of LP shares minted in the supply of the updated pool.
   string total_shares = 7;
+  // lp_fee: Total LP fees collected on this balance update, always in the BASE asset
+  string lp_fee = 8;
+  // bolt_fee: Bolt protocol fee collected on this balance update, always in the BASE asset
```

Comment:
```suggestion
  // bolt_fee: Bolt protocol fee collected on this balance update, always in the QUOTE asset
```
if you mean the protocol fee, that is always collected in QUOTE asset.

Reply:
Ahh, that was my lack of knowledge here.